zap/src/main/java/org/parosproxy/paros/core/scanner/HostProcess.java

```diff
@@ -640,6 +657,17 @@ private boolean canScanNode(StructuralNode node) {
             }
             return false;
         }
+        FilterResult filterResult = filterNode(node);
+        if (filterResult.isFiltered()) {
+            if (log.isDebugEnabled()) {
```

Comment:
This could be done in the other method, avoids the need to check here the result.

Reply:
Hi @thc202 , in which method ? already in the same method (canScanNode) we are logging multiple times if a node is filtered out.

Reply:
Log in `filterNode`.

Reply:
sure will do that.